packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -396,6 +396,7 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
       classificationAucRoc: `${ELASTIC_WEBSITE_URL}guide/en/machine-learning/${DOC_LINK_VERSION}/ml-dfa-classification.html#ml-dfanalytics-class-aucroc`,
       setUpgradeMode: `${ELASTICSEARCH_DOCS}ml-set-upgrade-mode.html`,
       trainedModels: `${ELASTIC_WEBSITE_URL}guide/en/machine-learning/${DOC_LINK_VERSION}/ml-trained-models.html`,
+      startTrainedModelsDeployment: `${ELASTIC_WEBSITE_URL}guide/en/elasticsearch/reference/${DOC_LINK_VERSION}/start-trained-model-deployment.html#start-trained-model-deployment-query-params`,
```

Comment:
Maybe it should be a nested object in this case?
```typescript
startTrainedModelsDeployment: {
  description: '...'
  queryParams: '...'
}
```
cc @szabosteve

Reply:
I don't _think_ you can use nested objects that deep in `get_doc_links.ts`. In case you don't want to experiment with it now @darnautov, it might be better to add `QueryParams` to the name.